Goal Creation and Goal

Anson-Bouchard opened the pull request:
The Goal branch includes the new Goal Creation page as well as the Goal page where users can view the goal they created.

vadymmusiienko commented:
Good work! I fixed a few things, so no action is needed on your end for this pull request.

Anson-Bouchard commented:
Ok thank you so much!

On Mon, Mar 24, 2025 at 10:11 AM Vadym Musiienko ***@***.***>
wrote:

> Good work! I fixed a few things, so no action is needed on your end for
> this pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vadymmusiienko/ubet/pull/10#issuecomment-2748844432>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGBPX5O5O44QUG5O3RNGC4L2WA4CTAVCNFSM6AAAAABZTTUBOGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBYHA2DINBTGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: vadymmusiienko]*vadymmusiienko* left a comment
> (vadymmusiienko/ubet#10)
> <https://github.com/vadymmusiienko/ubet/pull/10#issuecomment-2748844432>
>
> Good work! I fixed a few things, so no action is needed on your end for
> this pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vadymmusiienko/ubet/pull/10#issuecomment-2748844432>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGBPX5O5O44QUG5O3RNGC4L2WA4CTAVCNFSM6AAAAABZTTUBOGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBYHA2DINBTGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
